### PR TITLE
Fixed #50

### LINKED ISSUE #50: Create a keywords.txt firl

### PR DESCRIPTION
**Pull request by davidgatti**
@jedborovik check it out.

### CODE REVIEW

File: keywords.txt [hunk 1]
```diff
@@ -1,30 +1,100 @@
...
+setErrorHandler	KEYWORD2
 
-
-
-#######################################
+######################################
 # Constants (LITERAL1)
 #######################################
-
+KEY_NONE	LITERAL1
```
Comment (jedborovik) [inline keywords.txt]: what's `KEY_NONE`?

**Comment by jedborovik** [comment]
Methods look awesome. The only thing left that I can think of is `SerialPort1`, `SerialPort2`, and `SerialPort3`

File: keywords.txt (hunk 1 above)
Comment (davidgatti) [inline keywords.txt]: Who knows, it snook on me ;) I shall remove.

**Comment by davidgatti** [comment]
@jedborovik check my latest commit.

**Comment by emevarga** [comment]
Hell yeah this is dope.

**Comment by jedborovik** [comment]
Looks groovy to me :ship:

**Pull request closed without merging**